.env

```
@@ -1 +1,2 @@
-EXPO_PUBLIC_MEDPLUM_CLIENT_ID=848afde1-2521-4ef5-8b6d-4dc44ee7a304
+EXPO_PUBLIC_WEB_MEDPLUM_CLIENT_ID=264de708-e85f-4002-8e29-d6026a32efb6
+EXPO_PUBLIC_NATIVE_MEDPLUM_CLIENT_ID=848afde1-2521-4ef5-8b6d-4dc44ee7a304
```

Aside note: I noticed that the Medplum pattern for environment variables typically follows this pattern:

- `MEDPLUM_<NAME>_CLIENT_ID` 
- `MEDPLUM_<NAME>_CLIENT_SECRET`
- `MEDPLUM_<NAME>_ACCESS_TOKEN`

Based on that, those could be:

```suggestion
MEDPLUM_EXPO_PUBLIC_WEB_CLIENT_ID=264de708-e85f-4002-8e29-d6026a32efb6
MEDPLUM_EXPO_PUBLIC_NATIVE_CLIENT_ID=848afde1-2521-4ef5-8b6d-4dc44ee7a304
```

---

Also, I’m not sure if this directly affects the configuration for `react-native`, but for `react` apps using `Vite`, this pattern can help the configuration:

```ts
// vite.config.ts
// Reference: https://vite.dev/config/shared-options.html#envprefix

import ...

// https://vitejs.dev/config/
export default defineConfig({
  ...
  envPrefix: ['MEDPLUM_'], // THIS LINE - Specify custom environment variable prefix
  plugins: [react()],
  ...
});

```

---

EXPO_PUBLIC_ must be the prefix, but yes, could be called `EXPO_PUBLIC_MEDPLUM_WEB_CLIENT_ID`

---

> EXPO_PUBLIC_ must be the prefix

Oh, got it